statchatexpress/models/index.js

```diff
@@ -1,7 +1,7 @@
 import { User } from './user.js';
 import { Stats } from './stats.js';
 
-User.hasMany(Stats, { as: 'stats', foreignKey: 'userId' });
+User.hasOne(Stats, { as: 'stats', foreignKey: 'userId' });
```

Comment:
Was this an intentional change? I think the original was correct! This will be limiting later on if you decide to support multiple games.

Reply:
ok, i will change it back, i forgot that i was going to add more stats later.